clang/test/CheckedC/parsing/invalid-where-clause.c

```diff
@@ -37,6 +37,10 @@ void invalid_cases_nullstmt(_Nt_array_ptr<char> p, int a, int b) {
   _Where a _And p : _And q : count(0) _And x _And a = 1 _And f() < 0 _And; // expected-error {{invalid expression in where clause, expected bounds declaration or equality expression}} expected-error {{expected bounds expression}} expected-error {{use of undeclared identifier q}} expected-error {{use of undeclared identifier 'x'}} expected-error {{expected comparison operator in equality expression}} expected-error {{call expression not allowed in expression}} expected-error {{expected bounds declaration or equality expression in where clause}}
 }
 
+void invalid_cases_exprstmt(_Nt_array_ptr<char> p, int a) {
+  a = 0 _Where a _And p : _And q : count(0) _And x _Where b = 1 _And f() < 0 _And; // expected-error {{invalid expression in where clause, expected bounds declaration or equality expression}} expected-error {{expected bounds expression}} expected-error {{use of undeclared identifier q}} expected-error {{use of undeclared identifier 'x'}} expected-error {{use of undeclared identifier 'b'}} expected-error {{call expression not allowed in expression}} expected-error {{expected bounds declaration or equality expression in where clause}}
+}
+
```

Comment:
What is the error message emitted for an attempt to attach a where clause to statements other than expression statements? Perhaps a few test cases may help.